seed/challenges/basic-javascript.json

```diff
@@ -1077,8 +1077,9 @@
         "  return Math.random();",
```

Comment:
Is it correct to have a return statement here?  I don't see a function declaration?